can_spec_my18.yml

```diff
@@ -7,12 +7,8 @@ boards:
     arch: arm
     publish:
       can0:
-        - FrontCanNodeOutput
+        - FrontCanNodeBrakeThrottle
```

Comment:
Maybe just BrakeThrottle? Shorter names are better when you see that they end up in C code and cause long lines.